components/list/package.json

```diff
@@ -0,0 +1,46 @@
+{
+	"name": "@westpac/list",
+	"version": "0.0.1",
+	"description": "List component for the Westpac GEL",
+	"repository": "https://github.com/WestpacGEL/GEL.git",
+	"author": "Westpac GEL Team <gel@westpac.com.au>",
+	"contributors": [
+		{
+			"name": "Dominik Wilkowski",
+			"email": "Hi@Dominik-Wilkowski.com",
+			"url": "https://dominik-wilkowski.com"
+		},
+		{
+			"name": "Joss Mackison",
+			"email": "joss@thinkmill.com.au",
+			"url": "http://jossmackison.me"
+		},
```

Comment:
I've been removing Joss from the components we've built... but left Dom in there